springboard/tests/test_views.py

```diff
@@ -12,7 +14,7 @@ def setUp(self):
         self.workspace = self.mk_workspace()
         self.config = testing.setUp(settings={
             'unicore.repos_dir': self.working_dir,
-            'unicore.content_repo_url': self.workspace.working_dir,
+            'unicore.content_repo_urls': self.workspace.working_dir,
```

Comment:
Or does it need to be a string with `\n` at this point? Now I'm not sure how pyramid works.